.gitignore

```diff
@@ -28,6 +28,7 @@ coverage.lcov
 coverage_results.*
 htmlcov/
 monkeytype.*
+armicontrib-openmc
```

Comment:
Perhaps we could just replace this line with:

```
armicontrib-*
```

As a little future-proofing.